.contrib/Parser/DATAS/02 - Outdoor Zones/01 Kalimdor/Mount Hyjal/Molten Front/Quests.lua

```diff
@@ -295,6 +295,7 @@ _.Zones =
 						["coord"] = { 50.6, 87.3, 338 },
 						["sourceQuest"] = 29214,	-- The Shadow Wardens
 						["isBreadcrumb"] = true,
+						["isDaily"] = true,
```

Comment:
I'll believe that it's daily for now, even though it's not marked as such on Wowhead :)
I'm currently in the process of unlocking this, so will verify later.

Reply:
I believe this is removed from the game indeed, I just unlocked wardens and can't pick up this quest.

Reply:
It's a tough one.
- 2 sites have it marked as a daily: https://cata-twinhead.twinstar.cz/index.php?quest=29204 & https://www.wow-freakz.com/quest_helper.php?quest=29204&name=the-warden-s-charge
- One of said sites has it marked as "not available": https://cata-twinhead.twinstar.cz/index.php?quest=29204
- WoWhead & a guide on WoWhead mention it as a breadcrumb (so indeed, nog daily): https://www.wowhead.com/guide=58/a-guide-to-the-molten-front-and-respective-quests#molten-front-sections-the-shadow-wardens-area
- Blizzard API returns a 404
- Data For Azeroth doesn't have the quest listed (so it's either a daily, or not in the game)

Reply:
Marked as removed in 7fd71d53113fa5e4d4209e7fcd3518aaf3eff31f.